recipes/geolite2pp/all/conandata.yml

```diff
@@ -0,0 +1,9 @@
+sources:
+  "0.0.1-2561":
```

Comment:
@toge this recipe looks good to me, could you update this to the latest `2711` version before meging? Thanks! (Make sure to ping me so I don't miss it again!)

Reply:
@AbrilRBS 
I can't find latest (2711) version tarball. (https://www.ccoderun.ca/GeoLite2PP/download/?C=M;O=D)
Did you find a way to get lateset version?

Reply:
Oh I see, the site confused me, it's titled `v0.0.1-2711` but the latest released version is the current one, my bad! Sorry for the confussion